### PR TITLE
[plugin.video.filmarkivet@matrix] v3.3.0

### PR DESCRIPTION
**Pull request by Ikosse**
### Add-on update
* New maintainer: Ikosse
* Added context menu item for viewing video info
* Some performance optimization and code refactorization
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by basrieter** [comment]
@pugo Do you, as the original author, agree with this update from @Ikosse?

File: plugin.video.filmarkivet/tests/filmarkivet_unit.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
+import unittest
```
Comment (basrieter) [inline plugin.video.filmarkivet/tests/filmarkivet_unit.py]: Please don't include this file. It seems to be for testing only?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please remote the unit test file from your submission.

File: plugin.video.filmarkivet/tests/filmarkivet_unit.py (hunk 1 above)
Comment (Ikosse) [reply plugin.video.filmarkivet/tests/filmarkivet_unit.py]: Yes, that's true, I have removed the file now.

Thank you for the review!

**Comment by ronie** [comment]
@basrieter permission to adopt the plugin was granted here:
https://github-com.translate.goog/pugo/filmarkivet-xbmc/pull/3?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=nl&_x_tr_pto=ajax,elem#issuecomment-823896868

**Review by ronie — APPROVED** [review]
looks good to me, thx!

**Pull request merged**